Makefile

```diff
@@ -32,6 +32,7 @@ build:
 
 .PHONY: generate
 generate:
+	go install ./vendor/golang.org/x/tools/cmd/stringer
```

Comment:
specifically, this being missing causes make generate to fail. it works for folks who have manually installed stringer globally. this is a failure of golang to isolate projects.